packages/web-ui/src/lab/TextLink/TextLink.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import { ElementRef, forwardRef, PropsWithChildren } from 'react';
+import { PropsWithSx } from '../../types';
+import { getPrefixedName, px } from '../../utils';
+import clsx from 'clsx';
+import { Typography } from '../../Typography';
+import { TextLinkProps } from './TextLink.props';
+import { styled } from '../../theme';
+import { colors, colorsCommon } from '@utilitywarehouse/colour-system';
+import { useBackground } from '../../Box';
+
+const componentName = 'TextLink';
+const componentClassName = getPrefixedName(componentName);
+
+const StyledTypography = styled(Typography)(() => {
+  const { isBrandBackground } = useBackground();
+  return {
+    '--text-link-color-default': isBrandBackground ? colorsCommon.brandWhite : colors.cyan600,
+    '--text-link-color-active': isBrandBackground ? colorsCommon.brandWhite : colors.cyan800,
```

Comment:
I see that in the [colours](https://web-ui-git-rphoenix-uwds-1352-uwds.vercel.app/?path=/docs/colour-system-colors--documentation) white and midnight are not a color like on FIgma (screenshot), I wonder if we should use one source for the colors rather than the `brand` prefixed ones on this case. I know that there was a usage for these brand colours and common colors but maybe we should be using just `white` on this case

<img width="238" alt="Screenshot 2023-11-16 at 15 06 58" src="https://github.com/utilitywarehouse/design-systems/assets/1488514/392f5c25-98ad-4f37-927f-ec361c822fbd">